src/components/organisms/AdminDashboard/AdminDashboard.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useMemo, useState } from "react";
+import { Button, Dropdown, DropdownButton } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+
+import { isPartyMapVenue } from "types/venues";
+
+import { sortVenues, VenueSortingOptions } from "utils/venue";
+
+import { useOwnedVenues } from "hooks/useConnectOwnedVenues";
+
+import { AdminVenueCard } from "components/molecules/AdminVenueCard";
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import { AdminRestricted } from "components/atoms/AdminRestricted";
+
+import "./AdminDashboard.scss";
+
+export const AdminDashboard: React.FC = () => {
+  const { ownedVenues: venues, isLoading } = useOwnedVenues({});
+
+  const [
+    currentSortingOption,
+    setCurrentSortingOption,
+  ] = useState<VenueSortingOptions>(VenueSortingOptions.az);
+
+  const sortedVenues = useMemo(
+    () => sortVenues(venues, currentSortingOption) ?? [],
+    [currentSortingOption, venues]
+  );
+
+  const renderedPartyVenues = useMemo(
+    () =>
+      sortedVenues
+        ?.filter(isPartyMapVenue)
+        .map((venue) => <AdminVenueCard key={venue.id} venue={venue} />),
+    [sortedVenues]
+  );
+
+  const sortingOptions = useMemo(
+    () => (
+      <DropdownButton variant="secondary" title="Sort venues">
+        {Object.values(VenueSortingOptions).map((sortingOption) => (
+          <Dropdown.Item
+            key={sortingOption}
+            onClick={() => setCurrentSortingOption(sortingOption)}
+          >
+            {sortingOption}
+          </Dropdown.Item>
+        ))}
+      </DropdownButton>
+    ),
+    []
+  );
+
+  const hasVenues = renderedPartyVenues.length > 0;
+
+  if (isLoading) {
+    return <LoadingPage />;
+  }
+
+  return (
+    <AdminRestricted>
+      <div className="AdminDashboard">
+        <div className="AdminDashboard__header">
+          <div className="AdminDashboard__header--left">
+            <div className="AdminDashboard__header-title">Admin Dashboard</div>
+            {sortingOptions}
+          </div>
+          <Button as={Link} to="/admin-ng/create/venue">
+            Create a new space
+          </Button>
```

Comment:
[OOS]

A good use-case for upgrading `ButtonNG` to use the `linkTo` prop as a button

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2278/commits/324a563bac68a0febcb9913d6c546c19bbc68b04